test/api/core/notifier/notifyDocumentUpdate.test.js

```diff
@@ -65,7 +65,8 @@ describe('Test: notifier.notifyDocumentUpdate', () => {
           {
             _id,
             _meta: {canIhas: 'cheezburgers?'},
-            _source: {foo: 'bar'}
+            _source: {foo: 'bar'},
+            _updatedField: '_kuzzle_info'
```

Comment:
I didn't understand the test well I think, for me the field `foo` should be present in the `_updatedField` array no?
And here `_updatedField` is not an array

Reply:
Here is the request sent in this test :
```
request = new Request({
      controller: 'document',
      action: 'update',
      index: 'foo',
      collection: 'bar',
      _id: 'Sir Isaac Newton is the deadliest son-of-a-bitch in space',
      body: {
        _kuzzle_info: {
          'canIhas': 'cheezburgers?'
        },
        foo: 'bar'
      }
    });
```
The updated field is `_kuzzle_info` as you can see

In deed it's not an array, why should it be ?

Reply:
`_source` is the document, in the request it's updated by adding the field `_kuzzle_info`

Reply:
You cannot update the field `_kuzzle_info`, it's ignored by the Elasticsearch service.
I mean an array because an update request may modify several fields

Reply:
Right for the array
for the `_kuzzle_info` field : 
Since Kuzzle update `_kuzzle_info` is automatically updated, do I remove it from the resulted array or not ?
`['foo', '_kuzzle_info']` or
`['foo']`

Reply:
The `updatedFields` gives the following information: "the list of changed fields that triggered that real-time notification`.

`_kuzzle_info` is merely a side-effect, always updated alongside any update request, so it cannot be the cause of a real-time notification.

Ergo, `_kuzzle_info` should never appear in the `updatedFIelds` list.